src/index.js

```diff
@@ -3,54 +3,31 @@
   Author Tobias Koppers @sokra
 */
 
+import path from 'path';
```

Comment:
This can be removed, If these are applied:

https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/77#pullrequestreview-49008895
https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/77#pullrequestreview-49016047